### PR TITLE
TEP-0122: Mask as Implemented

### PR DESCRIPTION
**Pull request by chitrangpatel**
This PR marks the TEP as implemented.
See https://github.com/tektoncd/chains/issues/683 for reference.

### CODE REVIEW

File: teps/0122-complete-build-instructions-and-parameters.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-status: implementable
+status: implemented
```
Comment (jerop) [inline teps/0122-complete-build-instructions-and-parameters.md:2]: please add the implementation prs to the references section of the TEP

Comment (chitrangpatel) [reply teps/0122-complete-build-instructions-and-parameters.md:2]: Thanks Jerop! I added the references section.

**Review by wlynch — APPROVED** [review]
/lgtm

**Pull request merged**